daemon/internal/newrelic/integration/test.go

```diff
@@ -544,7 +544,12 @@ func (t *Test) compareMetricsExist(harvest *newrelic.Harvest) {
 				actualCount := int64(math.Round(actualData.([]interface{})[0].(float64)))
 
 				metricPasses := false
-				if (count == -1 && actualCount > 0) || (actualCount == count) {
+
+				// apdex metrics can have a count of 0 since the count field is
+				// actually the "satisfied" count, not a total count of metric
+				// as it is for other types of metrics
+				apdex_metric := strings.HasPrefix(expected, "Apdex/")
+				if (count == -1 && (apdex_metric || actualCount > 0)) || (actualCount == count) {
```

Comment:
Why not simply change `actualCount > 0` to `actualCount >= 0`?

Reply:
I did not want to match "normal" metrics with a total count of 0 - perhaps this is not desirable?

In the case of apdex metrics the "count" is not the total count as for normal metrics, but a count of times the apdex score was "satisfied", so 0 make sense.